core/src/main/java/com/sequenceiq/cloudbreak/repository/StackRepository.java

```diff
@@ -256,7 +256,11 @@
             + "u.userName as username, "
             + "u.userCrn as usercrn, "
             + "c.certExpirationState as certExpirationState "
-            + "FROM Stack s LEFT JOIN s.cluster c LEFT JOIN c.blueprint b LEFT JOIN s.stackStatus ss LEFT JOIN s.creator u "
+            + "FROM Stack s "
+            + "LEFT JOIN s.cluster c "
```

Comment:
this will cause heavy fetches from the database

Reply:
I have not modified the from clause, just made it more readable